packages/legend-application-query/src/__lib__/DSL_DataSpace_LegendQueryNavigation.ts

```diff
@@ -39,6 +39,7 @@ export enum DATA_SPACE_TEMPLATE_QUERY_CREATOR_ROUTE_PATTERN_TOKEN {
   GAV = 'gav',
   DATA_SPACE_PATH = 'dataSpacePath',
   TEMPLATE = 'template',
+  EXECUTION_CONTEXT = 'executionContext',
```

Comment:
dont think this is needed

Reply:
This is needed as when trying to build minimal graph, mapping and pmcd are fetched from corresponding sexecutionContext key

Reply:
Still don't understand. DataspaceInfo should have the template info( which we get from the template id) which has the executon context we need ? right

Reply:
This is the entry point of building DataspaceAnalyticsResult and light graph is built when building DataspaceAnalyticsResult.
Based on current implementation, template info is built at the last step